cpanfile

```diff
@@ -61,5 +61,5 @@ on 'test' => sub {
   requires 'Test::MockObject';
   requires 'Test::More', '0.88';
   requires 'Test::Warnings';
-  requires 'Test::YAML::Valid';
+  requires 'JSON::Validator';
```

Comment:
Do not forget to run `tools/update_spec` to update spec file.

Reply:
I did, but it did not change the spec file. Probably because it's only a test requirement.